book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rentals(rentals)
+    @rentals.push(rentals)
+    book.rentals = self
+  end
```

Comment:
- [ ] This method supposes to add a new rental and the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association, please fix that.
[Hint]: It will be like that:
```ruby
def add_rental(person, date)
    Rental.new(date, self, person)
end
```